### PR TITLE
remove postgresql as a dependency, close #249

### LINKED ISSUE #249: Remove Migrations Dependency On Postegresql
there is no reason for the helm migrations package to depend on postgresql as they're always installed in the context of dracon<br><br>there's also no reason for the postgresql connection string to be global

### PR DESCRIPTION
**Pull request by northdpole**
closes #249

### CODE REVIEW

File: deploy/deduplication-db-migrations/chart/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: deduplication-db-migrations
+description: |
+  A Helm chart for Kubernetes containing the dracon migrations needed for the deduplication component to run. 
+  Please check the documentation for more information
+type: application
+version: 0.7.0
+appVersion: 0.7.0
```
Comment (flowirtz) [inline deploy/deduplication-db-migrations/chart/Chart.yaml:8]: nit no empty newline at the end of the file

File: deploy/deduplication-db-migrations/chart/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: deduplication-db-migrations
+description: |
+  A Helm chart for Kubernetes containing the dracon migrations needed for the deduplication component to run. 
+  Please check the documentation for more information
```
Comment (flowirtz) [inline deploy/deduplication-db-migrations/chart/Chart.yaml:5]: can u maybe link to the relevant documentation so people know where to look?

File: deploy/deduplication-db-migrations/chart/templates/migrations-job.yaml [hunk 3]
```diff
@@ -30,6 +30,6 @@ spec:
         - migrations
         - apply
         - --url
-        - "postgresql://{{ .Values.global.postgresql.auth.username }}:{{ .Values.global.postgresql.auth.password }}@{{ .Values.global.postgresql.host }}/{{ .Values.global.postgresql.auth.database }}?{{ .Values.global.postgresql.auth.querystringargs}}"
-      serviceAccountName: {{ include "enrichment_db_migrations.fullname" . }}
+        - "postgresql://{{ .Values.database.auth.username }}:{{ .Values.database.auth.password }}@{{ .Values.database.host }}/{{ .Values.database.auth.database }}?{{ .Values.database.auth.querystringargs}}"
+      serviceAccountName: {{ include "deduplication_db_migrations.fullname" . }}
 {{ end }}
```
Comment (flowirtz) [inline deploy/deduplication-db-migrations/chart/templates/migrations-job.yaml:35]: no empty new line at end of file as well, there is a setting in vscode if u want to enable and forget about this: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

File: deploy/dracon/chart/Chart.yaml [hunk 4]
```diff
@@ -14,6 +14,6 @@ dependencies:
     version: 15.1.5
     repository: https://charts.bitnami.com/bitnami
     condition: mongodb.enabled
-  - name: enrichment-db-migrations
+  - name: deduplication-db-migrations
     version: 0.7.0
     repository: oci://ghcr.io/ocurity/dracon/charts
```
Comment (flowirtz) [inline deploy/dracon/chart/Chart.yaml:19]: see above

**Review by flowirtz — APPROVED** [review]
just some nits, didn't test this locally

File: deploy/dracon/chart/Chart.yaml
Comment (ptzianos) [inline deploy/dracon/chart/Chart.yaml:1]: this commit is not necessary, please drop it

File: deploy/deduplication-db-migrations/chart/Chart.yaml (hunk 2 above)
Comment (ptzianos) [reply deploy/deduplication-db-migrations/chart/Chart.yaml]: fixed

**Comment by ptzianos** [comment]
this will break the main branch for a few hours. because this is related to some hard deadlines, I will fix it tomorrow evening

**Review by dlicheva — APPROVED** [review]

**Pull request merged**